perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/ExpirationNotifScheduler.java

```diff
@@ -435,25 +433,21 @@ private void auditOldGroupExpirations(List<Status> allowedStatuses, Group group)
 	 * @throws InternalErrorException internal error
 	 */
 	private void auditIncomingExpirations(List<Status> allowedStatuses, Map<Integer, Vo> vosMap) throws InternalErrorException {
-		Calendar monthBefore = Calendar.getInstance();
-		monthBefore.add(Calendar.MONTH, 1);
+		LocalDate monthBefore = LocalDate.now().plusMonths(1);
```

Comment:
I would change the variable name. `monthBefore` is missleading. It should be something like `nextMonth`.

Reply:
Ok, it is fixed now, thank you.